Cython/Compiler/Main.py

```diff
@@ -61,16 +61,17 @@ class Context(object):
     #  the root of the module import namespace and the list
     #  of directories to search for include files.
     #
-    #  modules               {string : ModuleScope}
-    #  include_directories   [string]
-    #  future_directives     [object]
-    #  language_level        int     currently 2 or 3 for Python 2/3
+    #  modules                  {string : ModuleScope}
+    #  include_directories      [string]
+    #  future_directives        [object]
+    #  language_level           int     currently 2 or 3 for Python 2/3
+    #  legacy_implicit_noexcept [bool]
 
     cython_scope = None
     language_level = None  # warn when not set but default to Py2
 
     def __init__(self, include_directories, compiler_directives, cpp=False,
-                 language_level=None, options=None):
+                 language_level=None, legacy_implicit_noexcept=None, options=None):
```

Comment:
It might be easier to pick it out of `compiler_directives` in `Context.__init__` rather that having it as both a compiler directive and an option, and passing it as an argument to the constructor.

Reply:
I will have a look on that.

Reply:
I agree that it's not worth passing it around as call arguments.

Reply:
fixed.